option/option.go

```diff
@@ -146,8 +146,15 @@ var OperationID = fuego.OptionOperationID
 var Deprecated = fuego.OptionDeprecated
 
 // AddError adds an error to the route.
+// Deprecated: Use `AddResponse` instead.
```

Comment:
```suggestion
// Deprecated: Use [AddResponse] instead.
```

Godoc links should be preferred as they allow navigation in IDEs and in documentation